src/server.js

```diff
@@ -23,6 +25,11 @@ server.register([Inert,], (err) => {
     handler: (request, reply) => { reply.file('public/index.html'); },
   },
   {
+    path: '/login',
+    method: 'GET',
+    handler: (request,reply) => { reply.file('public/login.html'); },
```

Comment:
ops sorry I missed this
```js
hander: {

  file: {
    'public/login.html'
  }
}
```

but you can actually just make a single directory handler of this format for both the index and login html.

https://github.com/hapijs/inert

I think you can do something like set
`defaultExtention` to '.html'
and `index` to `true`
ie
```js
method: 'GET',
path: '/{param*}','.'
handler: {
  directory: {
    path: path.join(__dirname, /*fill this in*/),
    index: true,
    defaultExtension: '.html'
  }
}
```
I think also you current code will break if you call start from anywhare other than the root of your project (thats why I have added call to `path()`)

instead of doing it this way, the innert docs recomend
```js
const server = new Hapi.Server({
    connections: {
        routes: {
            files: {
                relativeTo: Path.join(__dirname, 'public')
            }
        }
    }
});
```